src/view/style.scss

```diff
@@ -139,13 +143,61 @@ $font__system: -apple-system, blinkmacsystemfont, 'Segoe UI', roboto, oxygen, ub
 		.newspack-popup-wrapper {
 			max-height: 100%;
 			overflow-y: auto;
-			width: 100%;
+			// width: 100%;
+		}
+
+		.newspack-popup {
+			// max-width: 90%;
+			padding-left: 0;
+			padding-right: 0;
+			// width: 1200px;
```

Comment:
Commented out code should be removed - here & in few other instances in this file.